tools/config/seed.config.ts

```diff
@@ -3,54 +3,241 @@ import { argv } from 'yargs';
 
 import { Environments, InjectableDependency } from './seed.config.interfaces';
 
+/**
+ * The enumeration of available environments.
+ * @type {Environments}
+ */
 export const ENVIRONMENTS: Environments = {
   DEVELOPMENT: 'dev',
   PRODUCTION: 'prod'
 };
 
-
+/**
+ * This class represents the basic configuration of the seed.
+ * It provides the following:
+ * - Constants for directories, ports, versions etc.
+ * - Injectable NPM dependencies
+ * - Injectable application assets
+ * - Temporary editor files to be ignored by the watcher and asset builder
+ * - SystemJS configuration
+ * - Autoprefixer configuration
+ * - BrowserSync configuration
+ * - Utilities
+ */
 export class SeedConfig {
+
+  /**
+   * The port where the application will run.
+   * The default port is `5555`, which can be overriden by the  `--port` flag
+   * when running `npm start`.
+   * @type {number}
+   */
   PORT                 = argv['port']                        || 5555;
```

Comment:
Also the question here is if we want to keep this indentation. it looked nice when all the constants were listed among each other. It might look a little weird now, but i guess thats just a question of style preference ;D